conans/test/functional/toolchains/cmake/test_shared_cmake.py

```diff
@@ -39,3 +43,156 @@ def test_shared_cmake_toolchain_test_package():
     client.save(files)
     client.run("create . -o hello:shared=True")
     assert "hello: Release!" in client.out
+
+
+@pytest.fixture()
+def test_client_shared():
+    client = TestClient()
+    files = pkg_cmake("hello", "0.1")
+    files.update(pkg_cmake_test("hello"))
```

Comment:
This might be the ``conan new cmake_lib`` template already?

Reply:
only changed the cpp file name, the output of the exe... half an hour debugging GRRRR!